.github/workflows/publish-auxiliary-packages.yml

```diff
@@ -58,3 +58,10 @@ jobs:
           npm config list
           echo "Publishing packages as $(npm whoami)"
           npm run publish-auxiliary
+
+      - name: "Publish tags"
```

Comment:
essentially same as https://github.com/mongodb-js/compass/blob/main/.github/workflows/publish-packages.yaml#L60 with one adjustment to not push tags for `mongosh` in this case